controllers/apply/form-router-next/joi-extensions/budget-items.js

```diff
@@ -37,6 +37,24 @@ module.exports = function budgetItems(joi) {
             }
         },
         rules: [
+            {
```

Comment:
You might not need this custom validator, as this extends array I think you can just call `.max` via `Joi.budgetItems().max(MAX_LIMIT)`

Reply:
Ahh yeah you're right, nice spot. Will change!